pkg/dashboard/resource/functiontemplate.go

```diff
@@ -56,16 +53,74 @@ func (ftr *functionTemplateResource) GetAll(request *http.Request) (map[string]r
 
 	for _, matchingFunctionTemplate := range matchingFunctionTemplates {
 
-		// add to attributes
-		attributes[matchingFunctionTemplate.Name] = restful.Attributes{
-			"metadata": matchingFunctionTemplate.Configuration.Meta,
-			"spec":     matchingFunctionTemplate.Configuration.Spec,
+		// if not rendered, add template in "values" mode, else just add as functionConfig with Meta and Spec
+		if matchingFunctionTemplate.FunctionConfigTemplate != "" {
+
+			attributes[matchingFunctionTemplate.Name] = restful.Attributes{
+				"template": matchingFunctionTemplate.FunctionConfigTemplate,
+				"values":   matchingFunctionTemplate.FunctionConfigValues,
+			}
+		} else {
+			renderedValues := make(map[string]interface{}, 2)
```

Comment:
No need to specify size, this isn't fast path. Just do `map[string]interface{}{}`